frontend/src/modules/teamcomments/components/TeamComment.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import TeamComment from './TeamComment';
+
+describe('<TeamComment>', () => {
+    const DEFAULT_USER = 'AndyDwyer';
+
```

Comment:
We should add a test case that the `CommentsList` is rendered if there are comments.